lib/SPIRV/libSPIRV/SPIRVDebug.h

```diff
@@ -40,6 +40,8 @@
 #ifndef SPIRVDEBUG_HPP_
 #define SPIRVDEBUG_HPP_
 
+#include "llvm/Support/raw_ostream.h"

```

Comment:
libSPIRV is designed to be LLVM independent so that it may be used by other applications.

How about abstract ostream by a macro spv_ostream and use conditional macro to define spv_ostream as std::ostream or llvm::raw_ostream ?

The same with spvdbgs().